examples/create-react-app/src/App.js

```diff
@@ -0,0 +1,74 @@
+import { Routes, Route, Link } from 'react-router-dom'
+import { Provider as RollbarProvider, ErrorBoundary, RollbarContext, useRollbar } from '@rollbar/react'
+
+import './App.css';
+
+const rollbarConfig = {
+  accessToken: process.env.REACT_APP_PUBLIC_ROLLBAR_TOKEN,
+  hostSafeList: [
+    'localhost:3000',
+    'localhost:4000',
+  ],
+  captureUncaught: true,
+  captureUnhandledRejections: true,
+  payload: {
+    environment: 'development',
+    client: {
+      javascript: {
+        code_version: '1.0.0',
+        source_map_enabled: true,
+      }
+    }
+  }
+};
+
+function App() {
+  return (
+    <RollbarProvider config={rollbarConfig}>
+      <ErrorBoundary>
```

Comment:
Might be a good opportunity to demo some of the props available on `ErrorBoundary`, though this is also fine as is.